Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ruby:2.6.3
+
+LABEL Name=rails_api_base Version=1.0
+
+RUN apt-get update -qq && apt-get install -y nodejs postgresql-client libxml2
+
+# Install heroku cli
+RUN curl https://cli-assets.heroku.com/install.sh | sh
+
+RUN mkdir /gems
+ENV BUNDLE_PATH "/gems"
+
+RUN mkdir /app
+WORKDIR /app
+
+# Install Node.js with npm
+RUN curl -sL https://deb.nodesource.com/setup_12.x | bash -
+RUN apt -y install nodejs
+RUN apt -y  install gcc g++ make cmake
+
+# Install yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt update && apt install yarn
+
+RUN apt install nano
+
+# Install rails globally
+RUN gem install rails -v 6.0.0
```

Comment:
Do we need this? 

I mean we already have a project with a Gemfile, wouldn't it be enough just to call `bundle install` once inside the project folder?

Reply:
The problem is, that if rails is not installed globally, we need to write bundle exec everytime we want to execute a rails command e.g.: `bundle exec rails db:create`. If rails is installed globally, this command turns into the usual one: `rails db:create`. I think this removes verboseness when using rails commands. What do you think?

Reply:
Don't we have the bin/rails command? I got the idea this command doesn't need the `bundle exec` 🤔

Reply:
I'd rather always use `bundle exec` then :D

Reply:
I'm not sure what would be the difference against using `bin/rails` vs `bundle exec` they seem pretty much the same, so whatever option would be fine for me

Reply:
The bin/rails executable is present, but you would have to cd into the bin directory. If that solution is okay with you, we can skip installing rails gem globally.

Reply:
So, the solution to this, was adding the the project's `bin` folder to the `PATH`. Now the commands present in that folder (such as rails, yarn, webpack), are globally accessible.